boards/raspberrypi/rpi_pico/doc/index.rst

```diff
@@ -153,6 +153,23 @@ devices as well as both PIO devices).
 Programming and Debugging
 *************************
 
+System requirements
+===================
+
+Prerequisites for the Pico W
+----------------------------
+
+Building for the RaspberryPi Pico W requires the AIROC binary blobs
```

Comment:
Not  a blocker:
```suggestion
Building for the Raspberry Pi Pico W requires the AIROC binary blobs
```

Reply:
Something needed to change, so this is going in.